actions/notify-workflow-completed/dist/config.json

```diff
@@ -21,6 +21,18 @@
         },
         "github.com/keep-network/tbtc-v2": {
             "workflow": "contracts.yml",
+            "downstream": [
+                "github.com/keep-network/keep-core/client"
+            ]
+        },
+        "github.com/keep-network/keep-core/client": {
+            "workflow": "client.yml",
+            "downstream": [
+                "github.com/threshold-network/token-dashboard"
+            ]
+        },
+        "github.com/threshold-network/token-dashboard": {
+            "workflow": "dashboard-ci.yml",
```

Comment:
At the moment the file we are reffering to is called `dashboard-ci.yaml`, but I'll rename it shortly to `dashboard-ci.yml`.

Reply:
https://github.com/threshold-network/token-dashboard/pull/152